test/DeadKeyTest.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Xunit;
+
+namespace Test
+{
+    public partial class ReadLine
+    {
+        [SkippableFact]
+        public void DeadKeyShouldBeIgnored()
+        {
+            Skip.If(this.Fixture.Lang != "fr-FR", "The dead key test requires Keyboard layout to be set to 'fr-FR'");
+            TestSetup(KeyMode.Cmd);
+
+            Test("aa", Keys("aa", _.DeadKey_Caret));
+            Test("aab", Keys("aa", _.DeadKey_Caret, 'b'));
+        }
```

Comment:
So this is just testing that the deadkey is ignored?

Can you also test a key that requires the deadkey?

Reply:
I would like to but not sure how to do it. With the French keyboard layout, when I press <kbd>[</kbd> then <kbd>q</kbd>, <kbd>[</kbd> will cause `Console.ReadKey` to return with a dead key, and then <kbd>q</kbd> will result in `<KeyChar = 'â', Key = A, Modifiers = 0>` returned by `Console.ReadKey`. In our tests, the mock console directly returns mocked `ConsoleKeyInfo` and doesn't cause any key actually be pressed, so there is no way to get the key `â` by mocking <kbd>[</kbd> + <kbd>q</kbd>.

I think we would need another mock console that can send keys to the `dotnet exec` process when `ReadKey` is called in order to test a key that requires the deadkey. Are you suggesting that or you have a different idea?

Reply:
I guess I was just thinking of changing your test to use something more like `<KeyChar = 'â', Key = A, Modifiers = 0>` as the second key. In other words, capture what `Console.ReadKey` actually sends us for our desired key sequences and make sure that gets rendered properly - don't try to emulate the actual key presses.

Reply:
OK, that sounds reasonable. I will make the change.

Reply:
Done, and tests passed with the French keyboard layout.

![image](https://user-images.githubusercontent.com/127450/59118472-1152ed80-8905-11e9-9ddb-1f77f735e30b.png)